libraries/AP_SmartAudio/AP_SmartAudio.cpp

```diff
@@ -0,0 +1,125 @@
+#include "AP_SmartAudio.h"
+#include "AP_SmartAudioDeviceV2_OneWay.h"
+#include <AP_HAL/AP_HAL.h>
+
+#define SMARTAUDIO_BAUDRATE 4800
+
+const AP_Param::GroupInfo AP_SmartAudio::var_info[] = {
+
+    // @Param: PROTOCOL
+    // @DisplayName: SmartAudio protocol version
+    // @Description: 0: None, 3: Version 2.0 One way
+    // @User: Standard
+    AP_GROUPINFO("PROTOCOL", 1, AP_SmartAudio, protocol_version, 0),
+
```

Comment:
Maybe worth add enabled param with ap_param_flag_enabled, so whole param list will not be visible to user what do not use smartaudio vtx.

Reply:
yes, good idea

Reply:
fixed